apps/bfd-pipeline/bfd-pipeline-rda-bridge/src/test/java/gov/cms/bfd/pipeline/bridge/RDABridgeIT.java

```diff
@@ -114,13 +116,16 @@ public void close() throws IOException {}
     FissClaimTransformer fissTransformer = new FissClaimTransformer(clock, hasher);
     McsClaimTransformer mcsTransformer = new McsClaimTransformer(clock, hasher);
 
+    RdaChange resultingTransform = null;
     for (MessageOrBuilder message : results) {
       if (message instanceof FissClaimChange) {
-        fissTransformer.transformClaim((FissClaimChange) message);
+        resultingTransform = fissTransformer.transformClaim((FissClaimChange) message);
       } else {
-        mcsTransformer.transformClaim((McsClaimChange) message);
+        resultingTransform = mcsTransformer.transformClaim((McsClaimChange) message);
       }
     }
+    // TODO: Add some better assertions here
+    assertNotNull(resultingTransform);
```

Comment:
I know you are just adding in something but shouldn't this go inside the loop at least? FYI @brianburton @KyRobbins

Reply:
Not sure, there were no assertions before so I just added this to ensure there was at least one. If someone has a suggestion, I can enhance.

Reply:
Given the explicit expected argument to `@Test` I think this was verifying that no exceptions were thrown.

Reply:
I think that was the idea, but Junit will already fail the test when any exception is thrown, so that isn't really a needed explicit assertion. I assume there is a state at the end of the functionality being tested here that we care about testing. Given its a transform, I made the assumption we at _least_ care that the transform returned something, and in calling it we can also test no exceptions got thrown to the end of the process.

Reply:
The intent of the test was to show that no exception is thrown.  It's true that the annotation is not required, but it is a cue to the reader as to why the tests exists, which is to check that no exception was thrown.  Checking that `resultingTransform` is not null is not the true intention of the test.